cluster/config-defaults.yaml

```diff
@@ -161,6 +161,7 @@ experimental_cluster_lifecycle_controller: "false"
 teapot_admission_controller_default_cpu_request: "25m"
 teapot_admission_controller_default_memory_request: "100Mi"
 teapot_admission_controller_process_resources: "true"
+teapot_admission_controller_ndots: "2"
```

Comment:
Let's keep it disabled for now, or at least enable only in test clusters first?

Reply:
Agreed, the reason why I added this was to run e2e tests to see if everything passes.